packages/genesys-spark-components/src/components/stable/gux-inline-alert/gux-inline-alert.tsx

```diff
@@ -50,10 +50,14 @@ export class GuxAlert {
           [`gux-${this.accent}`]: true
         }}
       >
-        <gux-icon icon-name={this.getIcon(this.accent)} decorative></gux-icon>
         <div class="gux-message-wrapper">
+          <gux-icon icon-name={this.getIcon(this.accent)} decorative></gux-icon>
           <div class="gux-sr-only">{this.i18n(this.accent)}</div>
-          <slot />
+          <div class="gux-content">
+            <slot name="content">
```

Comment:
I just wanted to make sure I understand the slot usage here. So we're slotting in the content slot and then below that the default slot? Maybe I'm misunderstanding. Thanks for the clarification.